cmd/topaz/main.go

```diff
@@ -173,9 +181,9 @@ func checkVersion(ctx *cc.CommonCtx) error {
 		ver.GetInfo().Version,
 	)
 	if !common.PromptYesNo("Do you want to update the configuration setting?", false) {
-		return nil
+		return false, nil
 	}
 
 	ctx.Config.Defaults.ContainerTag = ver.GetInfo().Version
-	return ctx.SaveContextConfig(common.CLIConfigurationFile)
+	return true, ctx.SaveContextConfig(common.CLIConfigurationFile)
```

Comment:
Since `ctx.Config.Defaults` is a singleton, would it make sense to update this as part of `SaveContextConfig`?